tests/josegonzalez/Dotenv/LoaderTest.php

```diff
@@ -141,6 +141,7 @@ public function testParse()
             'STRING_FALSE' => 'false',
             'STRING_EMPTY' => '',
             'STRING_EMPTY_2' => '',
+            '01SKIPPED'=>'skipped',
```

Comment:
Nah it will be for 3.0. 2.0 will be the release that I'm just about to release with all the updated stuff